### ISSUE TITLE
[en] Multiword chunker/EN_HybridDisambiguator, incorrect disambiguation: at any rate

### ISSUE THREAD

**Issue by MikeUnwalla** [languagetool-org/languagetool#7779]
> The flow can be set at any rate desired.

The word 'rate' is a noun, but the phrase 'at any rate' is disambiguated as a multi-word adverb:
![image](https://user-images.githubusercontent.com/5816160/215743903-53572fc1-9803-491d-bd7b-67512ecb8308.png)

**Comment by jaumeortola** [comment]
The multiword chunker is configured to remove all previous tags. If you need to keep the original tags, then remove "at any rate" from multiwords.txt, and add disambiguation rules as needed.

**Comment by MikeUnwalla** [comment]
@jaumeortola, thanks.

I added this rule at the end of disambiguation.xml:

```
    <rule name="at any rate" id="AT_ANY_RATE">
      <pattern>
        <token postag="RB">at</token>
        <token postag="RB">any</token>
        <token postag="RB">rate</token>
      </pattern>
      <disambig action="replace"><wd pos="IN"/><wd pos="DT"/><wd pos="NN:UN"/></disambig>
      <example type="ambiguous" inputform="rate[rate/RB]" outputform="rate[rate/NN:UN]">The flow can be set at any <marker>rate</marker> desired.</example>
    </rule>
```
The Token column in Tagger Result shows that the rule replaces the POS:
![image](https://user-images.githubusercontent.com/5816160/215753938-9216eb58-1e04-4306-af40-d10a89bd4a4c.png)

But, I get this testrules error:

`Exception in thread "main" java.lang.AssertionError: Disambiguated sentence is equal to the input sentence for rule: AT_ANY_RATE. The sentence was: <S> The[the/DT,B-NP-singular] flow[flow/NN:UN,E-NP-singular] can[can/MD,B-VP] be[be/VB,I-VP] set[set/VBN,I-VP] at[at/IN,at/RP,B-PP] any[any/DT,B-NP-singular] rate[rate/NN:UN,E-NP-singular] desired[desired/JJ,desire/VBD,desire/VBN,B-VP].[./.,</S>./PCT,O]`

The immediate solution is to remove the example from the rule.

I am leaving this issue open because despite the solution to my problem, my initial comment still applies. In the test sentence, the word 'rate' is a noun, not an adverb. (Feel free to close the issue if the incorrect analysis is not worth worrying about.)

**Comment by jaumeortola** [comment]
I would do this:
- Remove "at any rate" from multiwords.txt
- Create a disambiguation rule that tags "at any rate" as an adverb (if this tagging is needed), but with exceptions like "at any rate desired". 

On the other hand, it is possible to use the original word tags in rules with `<pattern raw_pos="yes">`.

**Comment by MikeUnwalla** [comment]
@jaumeortola, I agree with your suggestion. I will leave the changes to the LT team. (Somebody added 'at any rate' to multiwords.txt, so I guess that the RB analysis is/was wanted.)

**Comment by MikeUnwalla** [comment]
2023-02-21. I found some more:

- [ ] burning is VBG, not NN: I saw a cross **burning** fiercely on the hill.
- [ ] Notwithstanding is IN, not NN: **Notwithstanding** clauses of this nature, overall, the contract is good.
- [ ] magnifying is VBG, not NN: This type of microscope is necessary when you are **magnifying** glass fibres.
- [ ] above is IN, not RB: Also, **above** all the haze, the atmosphere is clear.
- [ ] same is JJ, not RB: The bolts must be all the **same** size.
- [ ] matter is NN, not RB: The story was accepted as a **matter** of fact, but it was a complete fiction.
- [ ] fairground is NN, not RB: I met her at the **fairground**.
- [ ] moment is NN, not RB: But at the **moment** I dropped the vase, the doorbell rang.
- [ ] before is IN, not RB: However, **before** long trousers for children became the norm, they wore short trousers.

Also, refer to the comments at the top of multiwords.txt.

Aside: line 1743 of multiwords.txt is:
`advisor 	NNP`
This is only 1 word, not two or more words, but Maven does not give a warning.
